src/machine/machine_atsamd51.go

```diff
@@ -1734,17 +1734,19 @@ func (pwm PWM) getMux() PinMode {
 
 // USBCDC is the USB CDC aka serial over USB interface on the SAMD21.
 type USBCDC struct {
-	Buffer  *RingBuffer
-	TxIdx   volatile.Register8
-	waitTxc bool
-	sent    bool
+	Buffer       *RingBuffer
+	TxIdx        volatile.Register8
+	waitTxc      bool
+	waitTxcTimer uint8
+	sent         bool
 }
 
 const (
 	usbcdcTxSizeMask uint8 = 0x3F
 	usbcdcTxBankMask uint8 = ^usbcdcTxSizeMask
 	usbcdcTxBank1st  uint8 = 0x00
 	usbcdcTxBank2nd  uint8 = usbcdcTxSizeMask + 1
+	usbcdcTxTimeout  uint8 = 5
```

Comment:
The timeout was set at about 5 ms.
It usually takes less than 1 ms to transmit.

Reply:
Perhaps this might be better named `usbcdcTxMaxRetriesAllowed` ?

Reply:
Thank you.
You're right, it's retry, not timeout.